mandatory/3-function-output.js

```diff
@@ -1,16 +1,21 @@
 // Add comments to explain what this function does. You're meant to use Google!
+
+
 function getRandomNumber() {
   return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
-  return word1.concat(word2);
+  
```

Comment:
Can you add a comment to explain what this function does (instead of changing the function?)

Reply:
Okay I will do that